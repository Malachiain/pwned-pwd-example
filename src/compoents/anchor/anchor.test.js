import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Anchor from './index'
describe('The anchor',()=>{
    it('It renders the anchor tag correctly',()=>{
        const wrapper = shallow(<Anchor href="test.com" conternt="test content"/>)
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})