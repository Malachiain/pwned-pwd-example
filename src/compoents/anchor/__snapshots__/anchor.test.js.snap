// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The anchor It renders the anchor tag correctly 1`] = `
<styled.a
  href="test.com"
  rel="noopener noreferrer"
  target="_blank"
/>
`;
